test/test_torch.py

```diff
@@ -18230,11 +18278,6 @@ def reference_implementation(res2):
     def test_cdiv(self, device, dtype):
         self._test_cop(torch.div, lambda x, y: x / y, dtype, device)
 
-    @onlyCPU
-    @dtypes(torch.float)
-    def test_cfmod(self, device, dtype):
-        self._test_cop(torch.fmod, math.fmod, dtype, device)
-
```

Comment:
Why delete test_cfmod? It looks like it is testing things that our test_fmod hasn't tested --  fmod(non-contiguous, non-contiguous)

Reply:
_test_cop can only be applied to torch.float and CPU. I am trying to move all the tests into test_fmod. I will add fmod(non-contiguous, non-contiguous) into `test_fmod` to make it complete.